package.json

```diff
@@ -27,6 +27,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@algolia/requester-node-http": "4.10.3",
     "@sentry/node": "6.5.1",
     "algoliasearch": "4.9.3",
```

Comment:
synchronise the requester and the client

Reply:
expose everything in algoliasearch 😬

Reply:
https://github.com/algolia/algoliasearch-client-javascript/pull/1284 among others I guess, but I agree, should be exposed :)

Reply:
<3 This will help us a lot on the crawler